bookmarks/models.py

```diff
@@ -123,6 +123,11 @@ def __init__(self, request: WSGIRequest):
         self.query = request.GET.get('q') or ''
         self.user = request.GET.get('user') or ''
 
+        query_date_keys = ['since_added', 'until_added', 'since_modified', 'until_modified']
```

Comment:
It took my brain a few turns to make sense of `until_added`, in general I think it would be easier to understand if the words were reversed.

How about:
- `added_since`
- `added_before`
- `modified_since`
- `modified_before`